lib/rbui/command/command_input.rb

```diff
@@ -40,8 +40,8 @@ def default_attrs
       {
         class: "flex h-10 w-full rounded-md bg-transparent py-3 text-sm outline-none placeholder:text-muted-foreground disabled:cursor-not-allowed disabled:opacity-50",
         placeholder: @placeholder,
-        data_action: "input->command#filter keydown.down->command#handleKeydown keydown.up->command#handleKeydown keydown.enter->command#handleKeydown keydown.esc->dismissable#dismiss",
-        data_command_target: "input",
+        data_action: "input->rbui--command#filter keydown.down->rbui--command#handleKeydown keydown.up->rbui--command#handleKeydown keydown.enter->rbui--command#handleKeydown keydown.esc->rbui--dialog#dismiss",
```

Comment:
@jacksonpires, we should isolate the JS controller. `rbui--dialog` is being referenced here. You should implement the dismiss action inside the command component.